subctl/issue-89-reconstructing-broker-info.md

```diff
@@ -0,0 +1,69 @@
+# Reconstruct broker-info.subm file from the running Broker
+
+Related Issue:
+[Option for recovering the broker-info.subm file from a running broker cluster](https://github.com/submariner-io/enhancements/issues/143)
+
+## Summary
+
+`broker-info.subm` file is created when a broker is installed. The file contains information critical to have a functional environment.
+There is a possibility that the file could be deleted or lost for one or the other reason. In such cases, no new clusters could be joined.
+
+This proposal describes a way to reconstruct the `broker-info.subm` file from the running broker.
+
+## Proposal
+
+A new `subctl` command will be added that would fetch needed data from the broker and write it to `broker-info.subm` file. The file
+would be Base64 encoded.
+
+## Design details
+
+* subctl command name: `reconstruct-brokerinfo`.
+* Flags for the command:
+  * `submconfig` to specify the cluster in which Submariner is installed
```

Comment:
Not sure I understand this. What exactly is this `submconfig`? and how is it related to `kubeconfig`? 
Does `submconfig` mean - SubmarinerConfig which is only applicable in OCM/ACM deployments?

Reply:
@sridhargaddam its a kubeconfig of a cluster which has Submariner installed.

Reply:
Changed the flag name to `submcluster`